x-pack/legacy/plugins/apm/public/components/shared/Summary/TransactionSummary.test.tsx

```diff
@@ -20,6 +20,21 @@ describe('TransactionSummary', () => {
       transaction
     };
 
+    it('renders', () => {
+      expect(() =>
+        shallow(<TransactionSummary {...props} />)
+      ).not.toThrowError();
+    });
+  });
+  describe('renders RUM transaction without request info', () => {
+    const transaction: Transaction = exampleTransactions.httpRumOK;
```

Comment:
I don't think you need the explicit type
```suggestion
    const transaction = exampleTransactions.httpRumOK;
```